task/buildah-remote/0.1/README.md

```diff
@@ -0,0 +1,37 @@
+# buildah-remote task
+
+This task is programmatically generated from the buildah task to keep it in sync, it should not be manually modified.
```

Comment:
It would be good to include the steps of how someone can regenerate this. Ideally, there would be a step on the CI that checks if it needs to be regenerated.

Reply:
I will create a follow up issue for this, I don't think we need it for the initial merge

Reply:
https://issues.redhat.com/browse/STONEBLD-1757